mlir/include/mlir/InitAllTranslations.h

```diff
@@ -20,6 +20,7 @@ namespace mlir {
 
 void registerFromLLVMIRTranslation();
 void registerFromSPIRVTranslation();
+void registerFromWasmTranslation();
```

Comment:
High-level question: Do we want `Wasm` throughout or perhaps `WasmSSA`, matching the dialect, <https://github.com/llvm/llvm-project/commit/a53489606a4e704e5a7ac148e950eabfe21397cd>?

Reply:
Just for clarity: it targets (at the moment) `wasmssa` dialect, but the import is already from Wasm binary files.